DuckDuckGo/SettingsViewController.swift

```diff
@@ -54,15 +68,11 @@ class SettingsViewController: UITableViewController {
         dateFilterText.text = UserText.forDateFilter(currentDateFilter())
     }
     
-    @IBAction func onDonePressed(_ sender: Any) {
-        dismiss(animated: true, completion: nil)
-    }
-    
     override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         if indexPath.section == 1 && indexPath.row == 0 {
             launchOnboardingFlow()
         }
-        if indexPath.section == 3 && indexPath.row == 0 {
+        if indexPath.section == 4 && indexPath.row == 0 {
```

Comment:
I've moved these into a struct of statics - I would normally go for an enum but it wasn't as elegant in this case.